api-reference/inbound-parse/validate-the-mx-record-setup-for-the-inbound-parse-functionality.mdx

```diff
@@ -4,6 +4,16 @@ openapi: get /v1/inbound-parse/mx/status
 
 <RequestExample>
 
+```javascript Node.js
+import fetch from 'node-fetch';
+
+const response = await fetch('https://api.novu.co/v1/inbound-parse/mx/status', {
+  method: 'GET',
+  
+});
+const data = await response.json();
+```
+
```

Comment:
This is not the Node SDK method for this. Don't we have that?

Reply:
No, the link you sent me didn't have it. And so, as you said, I copied it from the old docs.